bakar-dev.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Muhib ur Rahman Bakar",
+  "type": "personal",
+  "bio": "Principal Software Engineer | Web Developer | Javascirpt/Typescript | ReactJs | NodeJs | NextJs",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/bakardev",
+      "icon": "FaTwitter"
+    },
+     {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/bakardev/",
+      "icon": "FaInstagram"
+    },
+     {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/bakardev/",
+      "icon": "FaLinkedin"
+    },
+     {
+      "name": "Github",
+      "url": "https://github.com/bakar-dev",
+      "icon": "FaGithub"
+    },
```

Comment:
Also, remove the extra **'Comma'** from this line :+1: 
```Suggestion
    }
```

Reply:
Thanks. I have updated the file.